provider-infinispan/src/test/java/io/github/iprodigy/cache/provider/infinispan/InfinispanProviderTest.java

```diff
@@ -18,7 +18,8 @@ void putGetClearTest() {
             spec.provider(new InfinispanProvider());
             spec.maxSize(32L);
             spec.expiryTime(Duration.ofMinutes(1));
-            spec.removalListener((key, value, cause) -> log.info(key + ":" + value + "=" + cause));
```

Comment:
might have to promote this lambda to a full class so it can be annotated with `@Listener` & the method can be annotated with `@CacheStarted` (?)

Reply:
Looks like the fix was to just add the listener annotation on the class helper class `InfinispanListener`: https://github.com/iProdigy/cache-api/pull/2/commits/bd8c0470c0d0737f2d7a14d7be07b683cb01e1f5

Reply:
thank you, was confused with the spec lambda & the actual listener class